canonical-web-and-design/ubuntu.com
# 5008 maas automated hardware testing

## Done

Add maas takeover and landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/maas-hardware-testing](http://0.0.0.0:8001/engage/maas-hardware-testing) and  [http://0.0.0.0:8001/](http://0.0.0.0:8001/) 
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Refresh home page until maas takeover comes up.
- Make sure it complies with the [brief](https://docs.google.com/document/d/1xyNrWYsYcvux_gYtoWw39XD9cP4QmiAYJnDlLuV8CvE/edit?ts=5cab9e6f)
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5006)


## Issue / Card

Fixes #5008
